src/specs/VideoNativeComponent.ts

```diff
@@ -0,0 +1,536 @@
+/* eslint-disable @typescript-eslint/ban-types */
```

Comment:
Is it mandatory to move this file ?
It makes review harder ... (I think you remove and add instead of git mv)

Reply:
It's because of handling `{}` on codegen.
I wanted to use Readonly<object> but it cannot be possible.

But, I found another issue on OnReceiveAdEventData while I am looking this issue.
OnReceiveAdEventData['data'] type should be handled as array type I guess.
Because codegen cannot parse Record<string, string> type.
I think I just handle that type as just {} and I missed it.

Reply:
Or, we have to define OnReceiveAdEventData['data'] type as Readonly object if we can define what key and value are expected.

Reply:
@yungblud I think the point is that instead of deleting and adding a file (using `git add`), you should use `git mv` to move the file - this keeps the diff between files

Reply:
@KrzysztofMoch Thanks. That was my mistake.
I will try that!

Reply:
Hmm, I think I cannot do mv file clearly.
I tried multiple times but it's difficult to show just "mv" on file changes.